examples/k8s/otel-config.yaml

```diff
@@ -15,7 +15,7 @@ data:
           http:
     exporters:
       otlp:
-        endpoint: "otel-collector.default:55680" # TODO: Update me
+        endpoint: "otel-collector.default:4317" # TODO: Update me
```

Comment:
Is the TODO still needed? Or is it unrelated to the port itself?

Reply:
Removed. Thanks